content/id/docs/kegiatan-eastern-townhall-indonesia.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Kegiatan Eastern Townhall Indonesia"
+linkTitle: "Kegiatan Eastern Townhall Indonesia"
+hide_feedback: true
+weight: 20
+description:
+---
+
+# 
+
+Sebagai informasi, kegiatan ini khusus di Eastern Townhall (ETH) Indonesia saja dan dimulai sejak tanggal 10 Jan 2022
+
+> Program Community Engagement Bounty 1.0
```

Comment:
Use the headers instead of quotes syntax, use this ### instead of this '>'  quotes give the wrong context for what is its your trying to convey